types/strategy.ts

```diff
@@ -12,3 +12,23 @@ export interface Strategy {
   executePlugin: string;
   managePlugin: string;
 }
+
+export const strategySchema: Readonly<{
+  platform: (value: any) => boolean;
+  pair: (value: any) => boolean;
+  timeframe: (value: any) => boolean;
+  planPlugin: (value: any) => boolean;
+  executePlugin: (value: any) => boolean;
+  managePlugin: (value: any) => boolean;
+}> = {
+  platform: (value: any): boolean => value === 'dydx',
```

Comment:
Usage of `unknown` is usually preferred over `any` as it gives us more type safety.